### ISSUE TITLE
Throwbar started when changing sneak state quickly

### ISSUE THREAD

**Issue by RainbowDashLabs** [eldoriarpg/PickMeUp#6]
> if you make sneak sneak very quickly it drops the animal or the boat but the power gauge to throw continues to work while you can't throw anything.

**Comment by Gabo6480** [comment]
This also happens if you start the throwing process and before the gauge appears you rightclick to cancel it

**Comment by Gabo6480** [comment]
Also, if you pick up another mob and sneak for throwing this causes a "hyper-throw" which is insanely strong

**Comment by DaddyRobLIVE** [comment]
> Also, if you pick up another mob and sneak for throwing this causes a "hyper-throw" which is insanely strong

Yes! this also crashed my server a lot! I tried nerfing it by reducing the value of throw-force from the config file but it doesn't seem to change at all.
